src/main/java/prefs/Config.java

```diff
@@ -0,0 +1,8 @@
+package prefs;
+
+/**
+ * An interface that represents configuration information in HubTurbo.
+ */
+public interface Config {
```

Comment:
This is currently used in a similar way as Serializable (but without the guarantees and conventions). Still discussing if we need it.